string text
string language #Spanish: es, English: en  
---
bool success
